platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/sources/CustomGeometrySource.java

```diff
@@ -72,11 +75,10 @@ public void invalidateRegion(LatLngBounds bounds) {
    * in new requests to `GeometryTileProvider` for visible tiles.
    *
    * @param zoomLevel Tile zoom level.
-   * @param x Tile X coordinate.
-   * @param y Tile Y coordinate.
+   * @param x         Tile X coordinate.
+   * @param y         Tile Y coordinate.
    */
   public void invalidateTile(int zoomLevel, int x, int y) {
-    checkThread();
```

Comment:
The annotation is now wrong.  Since the class is annotated as UiThread, everything is UiThread only be default.  Need to add AnyThread annotation.  Also may want to mention that it is useable from any thread in the JavaDoc.